terraform/main.tf

```diff
@@ -114,6 +116,8 @@ module "aws_deploy-uat-eu-west-2" {
   providers = {
     aws = "aws.eu-west-2"
   }
+
+  depends_on = ["${module.aws_deploy-us-west-2.static_node_ips}"]
```

Comment:
From terraform/main.tf I see green as:
aws_deploy-us-west-2
aws_deploy-uat-eu-west-2

So this makes sense.